lib/init.coffee

```diff
@@ -6,13 +6,13 @@ module.exports =
     executablePath:
       title: 'Executable Path'
       type: 'string'
-      default: 'pep257'
-      description: "Path to executable pep257 cmd."
+      default: 'pydocstyle'
```

Comment:
I have something similar [here](https://github.com/AtomLinter/linter-php/blob/v1.1.8/lib/main.coffee#L11), but be warned users might get annoyed if they have the package installed but have no intention of using it on that machine (don't ask, long story...).